src/main/java/bio/terra/cli/context/GlobalContext.java

```diff
@@ -282,8 +285,22 @@ public CommandRunner getRunner(WorkspaceContext workspaceContext) {
    */
   @JsonIgnore
   public static Path getGlobalContextDir() {
-    // TODO: allow overriding this (e.g. env var != user home directory)
+    // default to the user's home directory
     Path parentDir = Paths.get(System.getProperty("user.home"));
+
+    // if the override environment variable is set and points to a valid directory, then use it
+    // instead
+    String overrideDirName = System.getenv(CONTEXT_DIR_OVERRIDE_NAME);
+    if (overrideDirName != null && !overrideDirName.isBlank()) {
+      Path overrideDir = Paths.get(overrideDirName).toAbsolutePath();
+      if (overrideDir.toFile().exists() && overrideDir.toFile().isDirectory()) {
+        parentDir = overrideDir;
```

Comment:
It would be good to check and bail if there's already a `.terra` directory or file under the override. I don't think we want to be merging contexts if we can avoid it.

Reply:
Every time you execute a CLI command, it's a new Java process that has to read the context file(s) into memory. So the majority of the time, I expect there will already be a `.terra` directory there and we want to reuse it.